platform/core/src/services/DicomMetadataStore/DicomMetadataStore.js

```diff
@@ -83,6 +83,25 @@ const BaseImplementation = {
       SeriesInstanceUID,
     });
   },
+  addSeries(seriesSummaryMetadata) {
```

Comment:
So if you follow this tree this really just setting the series summary?

We should call it as such, add series is kinda connfusing.